lib/internal/test_runner/harness.js

```diff
@@ -39,6 +39,9 @@ function createTestTree(rootTestOptions, globalOptions) {
     watching: false,
     config: globalOptions,
     coverage: null,
+    minLineCoverage: globalOptions.minimumLineCoveragePercent,
+    minBranchCoverage: globalOptions.minimumBranchCoveragePercent,
+    minFuncCoverage: globalOptions.minimumFunctionCoveragePercent,
```

Comment:
Please don't attach these to the harness object. In the place where you are using them in `test.js`, you can use `test.config. minimumLineCoveragePercent` and friends.